src/test/java/spoon/test/lambda/LambdaTest.java

```diff
@@ -458,6 +458,16 @@ public void testLambdaWithGenericExtendingMultipleInterfaces() {
 		assertIsWellPrinted("( elt) -> elt.test()", lambda);
 	}
 
+	@Test
+	public void testCastLambdaWithIntersection() {
+		final CtLambda<?> lambda1 = intersection.getElements(new TypeFilter<CtLambda<?>>(CtLambda.class)).get(1);
```

Comment:
could you add a one line comment stating the test intention in natural language:

    // contract: intersection types on lambda parameters are supported